core/src/main/java/hudson/model/queue/AbstractQueueSorterImpl.java

```diff
@@ -35,17 +35,13 @@ public int compare(BuildableItem lhs, BuildableItem rhs) {
      * sign(a-b).
      */
     protected static int compare(long a, long b) {
```

Comment:
This probably should be deprecated in favour of `Long.compare`

Reply:
I addressed your comments and marked the methods as deprecated and suggested Long.compare